server/boot/controller-boot.js

```diff
@@ -0,0 +1,30 @@
+var controller;
+
+module.exports = function(app) {
+  // here app.controller(ctrl) creates a controller wrapper to register
+  // the 'ctrl' class with strong-remoting
+  app.controller(ctrl).remoteMethod('speak', {
+    http: { path: '/', verb: 'get' },
+    returns: { arg: 'msg', type: 'string', root: true },
+  });
+};
+
+function ctrl(app, ctx) {
+  // actually ends up registering at /api/ctrl
+  this.name = 'TestName';
```

Comment:
See [this comment](https://github.com/strongloop/loopback/pull/2284/files#r63230483). 

We don't need to explicitly set the name but we need to set `ctrl.http.path` if we want to change the endpoint at which the controller is registered.